src/Confluent.Kafka/Error.cs

```diff
@@ -111,6 +149,17 @@ public Error(ErrorCode code, string reason)
         /// </summary>
         public bool IsFatal { get; }
 
+        /// <summary>
+        ///     Whether or not the error is retriable.
+        /// </summary>
+        public bool IsRetriable { get; }
+
+        /// <summary>
+        ///     Whether or not an open transaction is abortable
```

Comment:
Add note that this is only relevant for the transactional producer API.
And I would phrase it as 'current transaction' as not to add a new state to transactions nomenclature (Open).

Reply:
phrasing: yep.
i was thinking the note is redundant, but yep, might as well.